src/uct/api/uct.h

```diff
@@ -1105,6 +1105,8 @@ UCT_INLINE_API ucs_status_t uct_iface_flush(uct_iface_h iface, unsigned flags,
  * @param [in]    flags  Flags that control ordering semantic (currently
  *                        unsupported - set to 0).
  * @return UCS_OK         - Ordering is inserted.
+ *         UCS_ERR_NO_RESOURCE - Ordering cannot be inserted when the routine
+ *                               is called.
```

Comment:
@shamisp @yosefe I am not quite sure if I understand correctly here: UCS_ERR_NO_RESOURCE means the call tried to do a fence but the fence operation cannot be completed, user needs to call it again later. This happens when network (like ugni) cannot guarantee the ordering and needs to do a flush internally, and that flush is not completed. Is this correct?